6/ansible-6.build

```diff
@@ -48,30 +48,30 @@ community.routeros: >=2.0.0,<3.0.0
 community.sap: >=1.0.0,<2.0.0
 community.skydive: >=1.0.0,<2.0.0
 community.sops: >=1.2.0,<2.0.0
-community.vmware: >=2.2.0,<3.0.0
+community.vmware: >=2.3.0,<3.0.0
 community.windows: >=1.9.0,<2.0.0
-community.zabbix: >=1.5.0,<2.0.0
+community.zabbix: >=1.6.0,<2.0.0
 containers.podman: >=1.9.0,<2.0.0
 cyberark.conjur: >=1.1.0,<2.0.0
 cyberark.pas: >=1.0.0,<2.0.0
 dellemc.enterprise_sonic: >=1.1.0,<2.0.0
-dellemc.openmanage: >=5.2.0,<6.0.0
+dellemc.openmanage: >=5.3.0,<6.0.0
 dellemc.os10: >=1.1.0,<2.0.0
 dellemc.os6: >=1.0.0,<2.0.0
 dellemc.os9: >=1.0.0,<2.0.0
-f5networks.f5_modules: >=1.15.0,<2.0.0
+f5networks.f5_modules: >=1.16.0,<2.0.0
 fortinet.fortimanager: >=2.1.0,<3.0.0
 fortinet.fortios: >=2.1.0,<3.0.0
 frr.frr: >=1.0.0,<2.0.0
 gluster.gluster: >=1.0.0,<2.0.0
 google.cloud: >=1.0.0,<2.0.0
 hetzner.hcloud: >=1.6.0,<2.0.0
 hpe.nimble: >=1.1.0,<2.0.0
-ibm.qradar: >=1.0.0,<2.0.0
+ibm.qradar: >=2.0.0,<3.0.0
 infinidat.infinibox: >=1.3.0,<2.0.0
 infoblox.nios_modules: >=1.2.0,<2.0.0
 inspur.sm: >=2.0.0,<3.0.0
-junipernetworks.junos: >=2.10.0,<3.0.0
+junipernetworks.junos: >=3.0.0,<4.0.0
 kubernetes.core: >=2.3.0,<3.0.0
```

Comment:
Note:
```
WARNING: found collection dependency errors!
WARNING: community.okd version_conflict: kubernetes.core-2.3.1 but needs >=2.1.0,<2.3.0
```

Reply:
This must be restricted to < 2.3.0. Can you please fix it?

Reply:
There could be a new release of community.okd with improved support soon, let's chat about this at the community meeting tomorrow.

Reply:
The new version of community.okd did not make it in time. We're pinning k8s.core as a result.